python/mxnet/initializer.py

```diff
@@ -264,7 +264,22 @@ def __call__(self, name, arr):
 
 @register
 class Zero(Initializer):
-    """Initialize the weight to 0."""
+    """Initializes weights to zero. Note that this does not initialize the biases.
```

Comment:
Okay, I see that the base class initializes the biases to zero. I'll fix the doc.